persist/src/main/resources/db/migration/V5_1__UpdateAudit.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE audit
```

Comment:
Does this have to be a separate migration? There might be a way to roll back the other one locally and rerun it with this column included.

Reply:
Yea i was having trouble figuring that out but going to try to roll it back